routes.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const app = express();
+const port = 7000;
+
+const data = [
+  {
+    ID:1,
+    FoodItem1:"Chocolate",
+    FoodItem2:"Ketchup",
+    WeirdCombos:"Chocolate and Ketchup",
+    Description:"Eat dipped chocolate in ketchup."
+  },
+];
+
+app.use(express.json());
+
+app.get('/', (req, res) => {
+    res.json(data)
+});
+
+app.post('/', (req, res) => {
```

Comment:
Consider validating the request body in the POST and PUT requests. You can use a library like 'joi' for this purpose. This will ensure that the data inserted into the array has the correct format. [important]